python/ql/test/experimental/library-tests/frameworks/stdlib/FileSystemAccess.py

```diff
@@ -0,0 +1,7 @@
+open("filepath")  # $getAPathArgument="filepath"
+open(file="filepath")  # $getAPathArgument="filepath"
+
+o = open
+
+o("filepath")  # f-:$getAPathArgument="filepath"
+o(file="filepath")  # f-:$getAPathArgument="filepath"
```

Comment:
This could be mitigated with a bit of local flow (or even a type tracker), but I think it's veryr rare for people to actually alias `open`.